### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by backache** [openbenches/openbenches.org#324] [General]
I am quite intrerested in how the tagging mechanism works, for example, what is a valid tag, how can I explore the database via tags and how I can edit them.

Is there any documentation for openbenches, or perhaps a link to wiki so we can start to build one?

**Comment by backache** [comment]
I have found the one-pagers in github's "pages" format, if we were to create something bigger I know they suggest creating a documentation folder (https://pages.github.com/)

[CODE_OF_CONDUCT.md](https://github.com/openbenches/openbenches.org/blob/master/CODE_OF_CONDUCT.md)
[DEVELOPERS.md](https://github.com/openbenches/openbenches.org/blob/master/DEVELOPERS.md)
[README.md](https://github.com/openbenches/openbenches.org/blob/master/README.md)

**Comment by edent** [comment]
The tagging is... poor. It was a feature request. But it is rarely used. only 1,599 benches have any tags.

Documentation is at https://github.com/openbenches/openbenches.org#tagging

You can explore the database at https://github.com/openbenches/openbenches.org/blob/master/database/openbenc_benches_table_tag_map.sql

**Comment by backache** [comment]
I do think tags could be used as a way to further increase engagement with the content and add fun :-)

As far as increasing the number of tags, without writing frontend code some database analysis of the text could find some records and tags to add, for example "doctor", "cricket", four-year dates, month-names, people-names etc

As well as the direct relationship of a word in the text to a tag, it could lead to a tag that's is in itself a group of tags, for example anything tagged cricket is tagged sport, anything tagged doctor is tagged medical  

It could also lead to using an external source to show related content, for example, I have added a regex to wikidata so it understands the URL ob uses for tags, I then looked at the tag "poem" and had it automatically explain what a poem is.

![image](https://user-images.githubusercontent.com/6274724/210165570-cb3df9a9-85d1-4f9c-9b73-7b8b334cc00c.png)
